geomesa-accumulo/geomesa-accumulo-dist/pom.xml

```diff
@@ -54,31 +54,31 @@
         </dependency>
         <dependency>
             <groupId>org.locationtech.geomesa</groupId>
-            <artifactId>geomesa-blobstore-gs-plugin_${scala.binary.version}</artifactId>
+            <artifactId>geomesa-accumulo-process_${scala.binary.version}</artifactId>
```

Comment:
Git did a great job on this diff... only change here is `geomesa-process` -> `geomesa-accumulo-process`.